apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/live-preview-modal/style.scss

```
@@ -45,4 +46,7 @@
 
 .dialog__backdrop.live-preview-modal__overlay {
 	background-color: rgba(0, 0, 0, 0.7);
+	// Dialog sets `z-index: z-index("root", ".dialog__backdrop")`,
+	// but this modal is used outside of Calypso, so we need to set it manually.
+	z-index: 9999;
```

`z-index` was defined in https://github.com/Automattic/wp-calypso/blob/3292129/packages/components/src/dialog/style.scss#L16, 

I set this manually as live-preview-modal is out of Calypso and no way to use https://github.com/Automattic/wp-calypso/blob/3292129/client/assets/stylesheets/shared/functions/_z-index.scss#L153 (CMIIW).

---

Other components in `wpcom-block-editor-nux` are using `Modal` from `@wordpress/components` and don't have this issue.

I think we should use [Modal](https://github.com/WordPress/gutenberg/tree/7b8cb73fdfac5621a44e72f2aee3a4929b400023/packages/components/src/modal) rather than `Dialog`. cc: @fushar 

However, Dialog works fine in this PR with the z-index fixed.

I think it's safer to use `100000` because that is the z-index used for `.components-modal__screen-overlay`. See https://github.com/WordPress/gutenberg/blob/7b8cb73fdfac5621a44e72f2aee3a4929b400023/packages/base-styles/_z-index.scss#L116

```suggestion
	// Dialog sets `z-index: z-index("root", ".dialog__backdrop")`,
	// but this modal is used outside of Calypso, so we need to set it manually.
	z-index: 100000;
```

---

We can refactor, but I'd personally prefer we allocate our time to work on the smart modal later which will be done outside `wpcom-block-editor-nux` anyway (e.g. on `jetpack-mu-plugins`). Up to @okmttdhr  😄

---

Thank you for your comment, @miksansegundo @fushar!

>Other components in wpcom-block-editor-nux are using Modal from @wordpress/components and don't have this issue.
I think we should use [Modal](https://github.com/WordPress/gutenberg/tree/7b8cb73fdfac5621a44e72f2aee3a4929b400023/packages/components/src/modal) rather than Dialog. 

This sounds make sense. It'd be nice to use @wordpress/components in this kind of situation.
For now, I would not change it to use Modal, as this modal itself should be considered as an interim solution. https://github.com/Automattic/wp-calypso/issues/82187

>I think it's safer to use 100000 because that is the z-index used for .components-modal__screen-overlay. See [WordPress/gutenberg@7b8cb73/packages/base-styles/_z-index.scss#L116](https://github.com/WordPress/gutenberg/blob/7b8cb73fdfac5621a44e72f2aee3a4929b400023/packages/base-styles/_z-index.scss#L116)

I'll address this 👍